### ISSUE TITLE
Tooltips Condensed & Table - Accessibility

### ISSUE THREAD

**Issue by cathystoli** [chanzuckerberg/sci-components#705]
**Issue:**
1. Does not read with SR.

2. Cannot be utilized via keyboard navigation.

**Solution:**
1. Allow the component to be read and utilized via SR.

2. Allow component to be used via keyboard navigation.
